lib/src/generic/interceptors/csrf_interceptor.dart

```diff
@@ -19,6 +19,8 @@ import 'dart:async';
 import 'package:w_service/w_service.dart';
 import 'package:w_transport/w_transport.dart';
 
+List<String> csrfRequired = ['DELETE', 'PATCH', 'POST', 'PUT', 'TRACE'];
```

Comment:
Now that I see this, I don't think TRACE is required. @maxwellpeterson-wf can you confirm?